docs/src/pages/careers/lead-designer.md

```diff
@@ -85,14 +85,25 @@ You will collaborate with engineers, product managers, and other designers, to e
 
 ### What would be nice if you had, but isn't required
 
-- **Experience with open-source projects**. You have seen the particular product dynamics when in an OSS environment and are knowledgable of the challenges.
+- **Experience with open-source products**. You have seen the particular product dynamics when in an OSS environment and are knowledgable of the challenges.
```

Comment:
Not entirely sure, but I think that developers never refer to React as an open-source product, they use "open-source project".

```suggestion
- **Experience with open-source projects**. You have seen the particular product dynamics when in an OSS environment and are knowledgable of the challenges.
```